### PR TITLE
Remove mpc recovery

### PR DESCRIPTION
**Pull request by volovyks**
Since we are not planning to work on this code base - let's remove it. It should make future refactoring, building, and testing speed optimizations easier.
This PR does not include any optimizations in the project structure, let's do that in a separate PR.

### CODE REVIEW

File: mpc-recovery/DEPLOY.md [hunk 1]
```diff
@@ -1,86 +0,0 @@
-# Manually Deploying mpc-recovery to GCP
-
-## Requirements
```
Comment (volovyks) [inline mpc-recovery/DEPLOY.md:3]: @auto-mausx not sure if it is useful for us now

**Comment by auto-mausx** [comment]
Can we remove the running services? If so let me destroy the TF resources for MPC recovery first, then we can merge this and remove the terraform code.

**Comment by ppca** [comment]
> Can we remove the running services? If so let me destroy the TF resources for MPC recovery first, then we can merge this and remove the terraform code.

That is a good idea

**Review by ppca — APPROVED** [review]

**Review by ChaoticTempest — CHANGES_REQUESTED** [review]
since you moved the `config.toml` file to the parent dir, you need to remove one `..` so it's pointing to the right target folder

**Comment by volovyks** [comment]
@ChaoticTempest can you please point me at what path you are talking about?

**Comment by ChaoticTempest** [comment]
@volovyks actually nevermind, the git diff looked funky because it showed moving one file to another

**Review by ChaoticTempest — APPROVED** [review]

**Pull request merged**